### ISSUE TITLE
ME stocking input hatch does not work.

### ISSUE THREAD

**Issue by Lucas-Pardo** [MindBrain27/Sky-Greg#65]
I have tried a few recipes in a large chemical reactor using an ME stocking input hatch to supply the input fluids and it does not work. The multiblock is correctly formed, the hatch is properly connected (it says online), the UI indicates the required fluids are in the hatch just like in an interface but the LCR does not run any recipe. At first I thought it was due to the circuit even if I had the circuit in another input bus, but no circuit recipes don't work either. I have tried the stocking input bus (for items) and it does work. I guess this is another gregtech bug, do you know if there is a fix to this? I have searched for a while and I have not found anything.

**Comment by MindBrain27** [comment]
Will look into it, I need to play around with it a bit as I'm not too familiar with the stocking hatches.

**Comment by Lucas-Pardo** [comment]
By the way, I noticed that the only recipe for engine intake casings (the EV and IV version) involves crafting with tools, so you cannot autocraft them. Could you add an assembler recipe for them?

**Comment by Lucas-Pardo** [comment]
Also, none of the crafting recipes for the EXO-Craft mod are working. There is a single HV assembler recipe for every component (redstone circuit and the units) and none of them are working, I tried it on an HV and EV assembler.

**Comment by MindBrain27** [comment]
I'll look into the casings, I've been making secondary recipes for stuff like that as I go, and for EXO-Craft that's a strange one, some people have working recipes and some people do not but I'm guessing the first 2 slots of the recipe in the assembler look blank correct? there are supposed to be some GT wires as well but for some people they don't show. I think I need to change the EXO-Craft recipes to just use a crafting table.

**Comment by Lucas-Pardo** [comment]
Yes, you are right, the first two slots are blank for me and all units show the exact same recipe. Strange how that happens just for some people.

**Comment by MindBrain27** [comment]
![image](https://github.com/MindBrain27/Sky-Greg/assets/74363414/108cb1ae-d01e-4338-a4a4-cb43e465b473)

you should be seeing something like this, but the wires for some reason don't show up for some people, I believe even if you put the correct missing items in it still doesn't work but they are fine electrum and fine red alloy wires for the energy core and fine annealed copper and fine red alloy wires for the redstone circuit

**Comment by MindBrain27** [comment]
Ok so I fixed this (EXO-Craft) for next pack update, I changed how the items are called in the recipe and it should work for everyone now.

**Comment by Lucas-Pardo** [comment]
Nice, awesome work as always. By the way, did you figure anything about the stocking hatch? I don't really care anymore since I've been putting cuadruple hatches instead but I am just curious.

**Comment by MindBrain27** [comment]
![image](https://github.com/MindBrain27/Sky-Greg/assets/74363414/e8d61f2a-adbe-4c39-b93b-f06c9ea30113)
![image](https://github.com/MindBrain27/Sky-Greg/assets/74363414/d733fbc1-0df0-42ab-ab68-133d0d4c2ef0)
Both the Hatch and Bus seem to be working, If I set a filter it seems to stock the Hatch/Bus.

**Comment by MindBrain27** [comment]
I re read the original comment, these are not connected to a multiblock though, just testing the Hatch/Bus themselves, will test further.

**Comment by MindBrain27** [comment]
yeah you seem to be right they just don't want to work, everything looks like it should be working but it just doesn't run a recipe

**Comment by Lucas-Pardo** [comment]
Quick question, in the quest for the LCR you said that you can lock output hatches and buses using the corresponding filter (fluid or item filter) right? Is there any special way to do this because I have tried it and it does not seem to be working:

 
![image](https://github.com/MindBrain27/Sky-Greg/assets/107495790/52587b74-b4a4-4559-b025-191c38ef6926)

![image](https://github.com/MindBrain27/Sky-Greg/assets/107495790/93d700fe-f33d-4a7f-85dc-9c56537ae3e3)

Many LCRs that I had set up across several recipes are now broken because somehow today they decided to change the output hatch of every fluid and has literally broken half of my chemical production. I thought that it would always output to the same hatch, like a distillation tower, but it seems like that is not the case even though it went for many many hours (I'd say at least 20 hours) without ever doing this, that is what confuses me.

**Comment by MindBrain27** [comment]
Sorry I somehow didn't see that last post, I was messing around trying to get the output filters to work, have you tried setting the output to distinct with a fluid filter? I believe at the time of writing the quest it worked as intended but GT may have changed something at some point down the line. I can test later on but that's a suggestion off the top of my head.

**Comment by Lucas-Pardo** [comment]
Hi, no problem. Yeah, so I tried many things and was not able to get it to work, so I just dumped every chemical into an ME subnetwork. If you find a way though please tell me. On another topic, my energy production was based on combustion generators, so I used large combustion engines for EV and when I got to IV I was going to use the extreme combustion engine, the thing is that for it to be worth it (in my opinion) you need to boost it, at EV with oxygen and at IV with liquid oxygen. In 1.22 this was completely impossible because even though it says on the tooltip that it takes 80L/s (which I understand as 80mb/s) that was not the case and it seemed more like 80mb/t if not more, which is insane given that the only recipe for liquid oxygen takes 6s at IV, so you essentially need to waste all your energy to produce liquid oxygen to produce energy, it's stupid. Now in the latest modpack version I have not been able to try it because for some reason, even though the boost for the engine shows as 32768 EU/s, which is exactly 4A IV, it says that for me to be able to boost I need a bigger dynamo hatch, which should not be the case and as I said, it worked back in 1.22.
![image](https://github.com/MindBrain27/Sky-Greg/assets/107495790/f0fd5466-0641-4fa7-aab8-b82e48f5c10a)

By the way, this is still the only recipe for the intake casings, I hope you can add an assembler recipe:
![image](https://github.com/MindBrain27/Sky-Greg/assets/107495790/8429281c-875c-4ca6-82ab-a8168632bc33)

I kinda lost my will to play, I feel like the energy generation in IV is very limited, and I don't like building 20-30 EV generators just to keep progressing. I don't know, maybe you can recommend me some power generation method. I don't like the advanced generators, they are incredibly expensive compared to the extreme combustion generator and don't provide any advantage (in my opinion).
